sys/src/9/amd64/build.json

```diff
@@ -1,8 +1,16 @@
 [
 	{
 		"Name": "cpu",
+		"Pre": [
+		    "u-root -o uroot.cpio plan9",
```

Comment:
we need u-root in `bootstrap.sh`

Reply:
not sure what you mean.

Reply:
OH! now I know what you mean! 

OK I'll do that first, this PR is not quite right yet anyway, thanks!